org.openhab.binding.zigbee.telegesis/src/main/java/org/openhab/binding/zigbee/telegesis/internal/discovery/ZigBeeTelegesisUsbSerialDiscoveryParticipant.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2014-2018 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.zigbee.telegesis.internal.discovery;
+
+import static java.util.Arrays.asList;
+import static org.openhab.binding.zigbee.ZigBeeBindingConstants.CONFIGURATION_BAUD;
+import static org.openhab.binding.zigbee.ZigBeeBindingConstants.CONFIGURATION_PORT;
+import static org.openhab.binding.zigbee.telegesis.TelegesisBindingConstants.THING_TYPE_TELEGESIS;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.config.discovery.DiscoveryResult;
+import org.eclipse.smarthome.config.discovery.DiscoveryResultBuilder;
+import org.eclipse.smarthome.config.discovery.usbserial.UsbSerialDeviceInformation;
+import org.eclipse.smarthome.config.discovery.usbserial.UsbSerialDiscoveryParticipant;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * Discovery for ZigBee USB dongles, integrated in Eclipse SmartHome's USB-serial discovery by implementing
+ * a component of type {@link UsbSerialDiscoveryParticipant}.
+ * <p/>
+ * Currently, this {@link UsbSerialDiscoveryParticipant} supports three USB dongles built around
+ * a SiLabs EM357 ZigBee chip and a CP210x USB to UART bridge.
+ *
+ * @author Henning Sudbrock - initial contribution
+ */
+@Component(service = UsbSerialDiscoveryParticipant.class)
+public class ZigBeeTelegesisUsbSerialDiscoveryParticipant implements UsbSerialDiscoveryParticipant {
+
+    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = new HashSet<>(asList(THING_TYPE_TELEGESIS));
+
+    public static final int SILICON_LABS_USB_VENDOR_ID = 0x10c4;
+
+    /**
+     * For the {@link ZigBeeTelegesisUsbSerialDiscoveryParticipant#SILICON_LABS_USB_VENDOR_ID}, this USB product ID
+     * is used for the Telegesis ZigBee dongle model ETRX3USB+8M, as well as for one version of the QIVICON ZigBee stick
+     * (which is also produced by Telegesis).
+     */
+    public static final int QIVICON_DONGLE_V1_USB_PRODUCT_ID = 0x8293;
```

Comment:
I am a bit confused - the black Telegesis dongle is also supported, isn´t it? If yes shouldnt we rename the prefix to Telegesis?

Reply:
You are right - we could add TELEGESIS to the constant name (as this product ID is used both by one version of the Qivicon dongle, and by the dongle with Telegesis label). I will add this with my next commit to this PR.